Server/ConcordiaCurriculumManager/Controllers/DossierController.cs

```diff
@@ -51,4 +57,36 @@ public DossierController(IMapper mapper, ILogger<DossierController> logger, IUse
                 statusCode: StatusCodes.Status500InternalServerError);
         }
     }
+
+    [HttpPost(nameof(CreateDossierForUser))]
+    [SwaggerResponse(StatusCodes.Status400BadRequest, "Invalid input")]
+    [SwaggerResponse(StatusCodes.Status500InternalServerError, "Unexpected error")]
+    [SwaggerResponse(StatusCodes.Status201Created, "Dossier created successfully", typeof(Guid))]

```

Comment:
There's a mistake here, you don't return a typeof(GUID), you return a typeof(DossierDTO)

Reply:
Sounds good.